docs/protocol-methods.rst

```diff
@@ -1055,3 +1055,98 @@ Identify the client to the server and negotiate the protocol version.
 
   ["ElectrumX 1.2.1", "1.2"]
   "ElectrumX 1.2.1"
+
+masternode.announce.broadcast
+--------------
+
+Pass through the masternode announce message to be broadcast by the daemon.
+
+Whenever a masternode comes online or a client is syncing, they will send this message which describes the masternode entry and how to validate messages from it.
+
+**Signature**
+
+  .. function:: masternode.announce.broadcast(signmnb)
+
+  * *signmnb*
+
+    Masternode announce message.
+  
+**Result**
+
+  True if the announce was broadcasted succesfully otherwise False.
+
+masternode.subscribe
+--------------
+
+Returns the status of masternode.
+
+**Signature**
+
+  .. function:: masternode.subscribe(vin)
+
+  * *vin*
```

Comment:
We can do better than "vin".  how about "address"?  What does "collateral" mean?  Reading the parameter name and its description I have no idea what should be passed.